.github/workflows/docs.yaml

```diff
@@ -51,4 +51,4 @@ jobs:
         git config user.email "$EMAIL"
         git config user.name "$NAME"
         git commit -m "Automatic update for $GITHUB_SHA."
-        git push origin gh-pages
\ No newline at end of file
+        { git push origin gh-pages || true; }
```

Comment:
This line made the whole thing fail with "nothing to commit" when there were no changes to the gh-pages (no documentation update)